manifests/system-podsecuritypolicy.yml

```diff
@@ -0,0 +1,91 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: default-psp
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
+spec:
+  privileged: true
+  allowPrivilegeEscalation: true
+  allowedCapabilities:
+  - '*'
+  volumes:
+  - '*'
+  hostNetwork: true
+  hostPorts:
+  - min: 0
+    max: 65535
+  hostIPC: true
+  hostPID: true
+  runAsUser:
+    rule: 'RunAsAny'
+  seLinux:
+    rule: 'RunAsAny'
+  supplementalGroups:
+    rule: 'RunAsAny'
+  fsGroup:
+    rule: 'RunAsAny'
+
+---
+# Gives it's binding the ability to use the
+# specified default-psp.
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: default-psp-role
+rules:
+- apiGroups: ['policy']
+  resources: ['podsecuritypolicies']
+  verbs:     ['use']
+  resourceNames:
+  - default-psp
+
+---
+# Gives Helm the ability to create pods in
```

Comment:
Shouldn't we either bind this to just Helm, or note in the comment that it's authorizing the entire default system:serviceaccounts group?

Reply:
Open to suggestions on best approach.

Reply:
Since the Helm Controller jobs run using a ServiceAccount per Job, and we are restricted to using the built-in system:serviceaccounts groups (can't add ServiceAccounts to custom groups), this is the best we can do without modifying the HelmController itself to manage PSPs

```suggestion
# Gives all ServiceAccounts the ability to create pods in
```